platform/ext/target/stm/common/scripts/postbuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -
+#!/usr/bin/env bash
```

Comment:
I'm fine with this, although do we know if the included script uses any bash specific features? Otherwise just `#!/bin/sh` could be fine.

Reply:
I didn't check it; it could be checked but either way it'd be another patch.

Reply:
No; I did this here directly.

Reply:
Can you please submit this PR to the TF-M upstream repository?
We can cherry-pick it to the zephyr fork once it is merged upstream.

Have a look here for how to do it:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/tfm/docs/contributing/contributing_process.html